File: App/app/src/main/java/com/example/patientrecord/Classes/LieuThuoc.java
package com.example.patientrecord.Classes;

public class LieuThuoc {
    public int iD_BUOI_KHAM;
    public String iD_THUOC;
    public int lieU_SANG;
    public int lieU_TRUA;
    public int lieU_CHIEU;
    public int lieU_TOI;

    public LieuThuoc(String id_thuoc, int lieu_sang, int lieu_trua, int lieu_chieu, int lieu_toi){
        this.iD_THUOC = id_thuoc;
        this.lieU_SANG = lieu_sang;
        this.lieU_TRUA = lieu_trua;
        this.lieU_CHIEU = lieu_chieu;
        this.lieU_TOI = lieu_toi;
    }
}

File: API/API_MEDICAL/Entities/BuoiKham/LieuThuoc.cs
using Microsoft.EntityFrameworkCore;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace API_MEDICAL.Entities.BuoiKham
{
    [Table("LIEU_THUOC", Schema="patient_record")]
    [PrimaryKey("ID_BUOI_KHAM", "ID_THUOC")]
    public class LieuThuoc
    {
        [ForeignKey("BuoiKham")]
        [Required]
        public int ID_BUOI_KHAM { get; set; }
        [ForeignKey("Thuoc")]
        [Required]
        public string ID_THUOC { get; set; }
        public int LIEU_SANG { get; set; }
        public int LIEU_TRUA { get; set; }
        public int LIEU_CHIEU { get; set; }
        public int LIEU_TOI { get; set; }
    }
}
